scintellometry/io/arochime.py

```diff
@@ -28,6 +28,10 @@ def __init__(self, raw_files, blocksize,
         real and imaginary for each timestamp.
         """
         self.meta = eval(open(raw_files[0] + '.meta').read())
+#        meta_last = eval(open(raw_files[-1] + '.meta').read())
```

Comment:
These things are a good idea, but best for a separate pull request -- could you remove the lines here?